CHANGELOG.md

```diff
@@ -2,7 +2,54 @@
 
 ## [Unreleased](https://github.com/CosmWasm/wasmd/tree/HEAD)
 
-[Full Changelog](https://github.com/CosmWasm/wasmd/compare/v0.32.0...HEAD)
+[Full Changelog](https://github.com/CosmWasm/wasmd/compare/v0.40.0...HEAD)
+
+## [v0.40.0](https://github.com/CosmWasm/wasmd/tree/v0.40.0) (2023-05-2x)
+
+[Full Changelog](https://github.com/CosmWasm/wasmd/compare/v0.32.0...v0.40.0)
+
+- Import export simulation test for `x/wasm` is missing [\#1372](https://github.com/CosmWasm/wasmd/issues/1372)
+- Better tracking of CosmWasm capabilities [\#1341](https://github.com/CosmWasm/wasmd/issues/1341)
+- Rename `lastIDKey` key [\#1182](https://github.com/CosmWasm/wasmd/issues/1182)
+- Use ICS4Wrapper to send raw IBC packets & fix Fee middleware in wasm stack \(backport \#1375\) [\#1379](https://github.com/CosmWasm/wasmd/pull/1379)
+- Add wasm store to import-export sims [\#1374](https://github.com/CosmWasm/wasmd/pull/1374)
+- Bumped SDK to 0.47.2 and CometBFT to 0.37.1 [\#1369](https://github.com/CosmWasm/wasmd/pull/1369)
+- Remove starport config [\#1359](https://github.com/CosmWasm/wasmd/pull/1359)
+- Proper v1 gov support for wasm msg types [\#1301](https://github.com/CosmWasm/wasmd/issues/1301)
+- Cleanup `ErrNotFound` cases [\#1258](https://github.com/CosmWasm/wasmd/issues/1258)
+- New proto annotations  [\#1157](https://github.com/CosmWasm/wasmd/issues/1157)
+- Simulations with '--dry-run' return an error [\#713](https://github.com/CosmWasm/wasmd/issues/713)
+- Add wasmvm decorator option [\#1348](https://github.com/CosmWasm/wasmd/pull/1348)
+- More verbose error message [\#1354](https://github.com/CosmWasm/wasmd/pull/1354)
+- Remove gogo/protobuf from the 47 build's dependencies [\#1281](https://github.com/CosmWasm/wasmd/issues/1281)
+- Set final ibc-go version [\#1271](https://github.com/CosmWasm/wasmd/issues/1271)
+- Upgrade to cosmos-sdk proto 0.47.x [\#1148](https://github.com/CosmWasm/wasmd/issues/1148)
+
+### Notable changes:
```

Comment:
Notable changes covers lots of key points.

It would be good to add a line at the top (right after `Full Changelog`) saying something like:

"""
Wasmd 0.40 has a large dependency upgrade of the Cosmos SDK version from 0.45 to 0.47. Please read notable changes below to learn more.
"""